examples/examples.rst

```diff
@@ -0,0 +1,439 @@
+for which a cold air bubble (of radius :math:`r_c`) drops by convection in a neutrally
+stratified atmosphere. Its initial condition is defined in terms of the Exner pressure,
+:math:`\pi(\boldsymbol{x},t)`, and potential temperature,
+:math:`\theta(\boldsymbol{x},t)`, that relate to the state variables via
+
+.. math::
+    \rho &= \frac{P_0}{( c_p - c_v)\theta(\boldsymbol{x},t)} \pi(\boldsymbol{x},t)^{\frac{c_v}{ c_p - c_v}} \, ,
+
+    e &= c_v \theta(\boldsymbol{x},t) \pi(\boldsymbol{x},t) + \boldsymbol{u}\cdot \boldsymbol{u} /2 + g z \, ,
+
+where :math:`P_0` is the atmospheric pressure. For this problem, we have used no-slip
+and non-penetration boundary conditions for :math:`\boldsymbol{u}`, and no-flux
+for mass and energy densities. This problem can be run with::
+
+   ./navierstokes -problem density_current
+
+
+Bakeoff Problems
+======================================
+
+The Center for Efficient Exascale Discretizations (CEED) uses Bakeoff Problems (BPs)
+to test and compare the performance of high-order finite element implementations. The
+definitions of the problems are given on the ceed
+`website <https://ceed.exascaleproject.org/bps/>`_. Each of the following bakeoff
+problems that use external discretization libraries (such as MFEM, PETSc, and Nek5000)
+are located in the subdirectories :file:`examples/mfem`, :file:`examples/petsc`, and
+:file:`examples/nek5000`, respectively.
+
+Here we provide a short summary:
+
++-------------------------+----------------------------------------------------------------+
+| User code               | BPs                                                            |
++-------------------------+----------------------------------------------------------------+
+|                         | - BP1 (scalar mass operator), with :math:`q=p+2`               |
+| ``mfem``                | - BP3 (scalar Laplace operator), with :math:`q=p+2`            |
++-------------------------+----------------------------------------------------------------+
+|                         | - BP1 (scalar mass operator), with :math:`q=p+2`               |
+|                         | - BP2 (vector mass operator), with :math:`q=p+2`               |
+|                         | - BP3 (scalar Laplace operator), with :math:`q=p+2`            |
+| ``petsc``               | - BP4 (vector Laplace operator), with :math:`q=p+2`            |
+|                         | - BP5 (collocated scalar Laplace operator), with :math:`q=p+1` |
+|                         | - BP6 (collocated vector Laplace operator), with :math:`q=p+1` |
++-------------------------+----------------------------------------------------------------+
+|                         | - BP1 (scalar mass operator), with :math:`q=p+2`               |
+| ``nek5000``             | - BP3 (scalar Laplace operator), with :math:`q=p+2`            |
++-------------------------+----------------------------------------------------------------+
```

Comment:
We should fix this
BP1/2/3/4 we have Q = P+1 (which is degree+2)
BP 5/6 we have Q=P
I made the above mistake (Q=P+2) when I first coded this up in PETSc

Reply:
Thanks. The website should be updated then, because I was referring to it
https://ceed.exascaleproject.org/bps/

Reply:
I think there is a difference in convention. They are using P as polynomial order while use it to mean number of basis nodes.

Reply:
They say "The nodal points, denoted by p, are GLL (Gauss-Legendre-Lobatto)"
so they seem to be using small p for the nodes, i.e., one more than the degree of the polynomial. And that was the previous version that in fact I copied from the website